### PR TITLE
Option for disabling weekends

### PR DESCRIPTION
**Pull request by cupofjoakim**
I added an option for disabling weekends as per Issue [https://github.com/dbushell/Pikaday/issues/83].

### CODE REVIEW

**Comment by ppiekarczyk** [comment]
Any updates on this or #82 @cupofjoakim

**Comment by cupofjoakim** [comment]
@ppiekarczyk I had totally forgotten about this pull request. I think @bjbrewster has a point, and I agree. It'd just be pretty nice for the end user to have a quick way of disabling weekends all in all, since it's probably something many want to do (businesses being generally closed on weekends, etc).

**Comment by ppiekarczyk** [comment]
@bjbrewster has a good point, but this PR has been stale for almost a year. Can we incorporate weekends first and then move towards #82 as the catch-all for something like this?

Is that something we can agree on amongst those involved?

File: pikaday.js [hunk 1]
```diff
@@ -102,6 +102,19 @@
         return (/Date/).test(Object.prototype.toString.call(obj)) && !isNaN(obj.getTime());
     },
 
+    isWeekend = function(value, opt)
+    {
+        if(opt){
+            return false;
+        }
+        var n = value.getDay();
+        if ((n === 0) || (n === 6)) {
```
Comment (cupofjoakim) [inline pikaday.js]: That is a really good question. It seems like there's no reason.

**Comment by rikkert** [comment]
Fixed by merging #171

**Pull request closed without merging**